account_default_draft_move/account.py

```diff
@@ -27,8 +27,12 @@ class AccountInvoice(models.Model):
     def action_move_create(self):
         """Set move line in draft state after creating them."""
         res = super(AccountInvoice, self).action_move_create()
+        use_journal_setting = bool(self.env['ir.config_parameter'].get_param(
+            'use_journal_setting', default=False))
```

Comment:
use_journal_settings is perhaps difficult to understand when you look at the list of system parameters. The name of the parameter could be more explicit.